_posts/2023-02-13-homebrew-4.0.0.md

```diff
@@ -0,0 +1,54 @@
+- [`brew install --adopt` allows taking ownership of existing installed casks](https://github.com/Homebrew/brew/pull/14033).
+- [`brew install --dry-run` allows viewing what `brew install` will do before it is run](https://github.com/Homebrew/brew/pull/13706).
+- [`brew docs` opens](https://github.com/Homebrew/brew/pull/13844) [docs.brew.sh](https://docs.brew.sh).
+- [`--cask` is never required on Linux](https://github.com/Homebrew/brew/pull/14206).
+- [`service do` blocks allow defining a `run` command per platform](https://github.com/Homebrew/brew/pull/14371).
+- [`brew install` uses the local cache while installing dependencies from `pip`](https://github.com/Homebrew/brew/pull/14239).
+- [`brew doctor` no longer complains about BitDefender](https://github.com/Homebrew/brew/pull/14300).
+- [`brew install` will also suggest casks rather than just formulae when it fails to find the requested package](https://github.com/Homebrew/brew/pull/14240).
+- [`brew readall` simulates all architecture and OS configurations for better reliability](https://github.com/Homebrew/brew/pull/13990).
+- [`git` partial clones with sparse checkouts are supported when downloading from `git`](https://github.com/Homebrew/brew/pull/14035).
+- [GCC 12 is used for runtime libraries to match Ubuntu 22.04, where we build our bottles](https://github.com/Homebrew/brew/pull/13882).
+- [`brew doctor --quiet` prints no output on success](https://github.com/Homebrew/brew/pull/14515).
+- [`brew gist-logs` better detects missing permissions](https://github.com/Homebrew/brew/pull/14523).
+- [`brew update` uses the GitHub API token if available to avoid hitting rate limits](https://github.com/Homebrew/brew/pull/14349).
+- [We have discontinued WSL 1 support and recommend WSL 2 instead](https://github.com/Homebrew/brew/pull/14000).
+- [`brew fetch` and `brew install` can automatically determine mirrors for `glibc-bootstrap` and PyPI resources](https://github.com/Homebrew/brew/pull/13875).
+
+Finally:
+
+- Homebrew is now on Mastodon: [@homebrew@fosstodon.org](https://fosstodon.org/@homebrew).
+- [Our governance documents have been updated as part of our AGM](https://github.com/Homebrew/brew/pull/14482).
+- [We have switched from HackerOne](https://github.com/Homebrew/brew/pull/14132) to [GitHub for vulnerability reporting](https://github.com/homebrew/brew/security/advisories/new).
+- [Homebrew accepts donations through GitHub Sponsors](https://github.com/sponsors/Homebrew) and [still accepts donations through Patreon](https://www.patreon.com/homebrew). If you can afford it, please consider donating. If you'd rather not use GitHub Sponsors or Patreon (our preferred donation methods), [check out the other ways to donate in our README](https://github.com/homebrew/brew/#donations).
+
+Thanks to all our hard-working maintainers, contributors, sponsors and supporters for getting us this far.
```

Comment:
Might this be a cool opportunity to post that picture from FOSDEM?

Reply:
@colindean you mean to further humiliate me in that stupid costume? I THINK NOT

(ok I'll add it)

Reply:
🤣